tests/unit_tests/test_ladder_service.py

```diff
@@ -217,8 +217,9 @@ async def test_start_game_timeout(
         mock.call({
             "command": "notice",
             "style": "info",
-            "text": "You have received 1 violations for failing to connect "
-            "to matchmaker games."
+            "text": "You have caused a matchmaking connection failure 1 time(s). "
+                    "Multiple failures result in temporary time-outs from matchmaker. "
+                    "Please seek support on the forums or discord for persistent issues."
```

Comment:
Can you format the multiline strings like this.
```suggestion
            "text": (
                "You have caused a matchmaking connection failure 1 time(s). "
                "Multiple failures result in temporary time-outs from matchmaker. "
                "Please seek support on the forums or discord for persistent issues."
            )
```